ThirdParty/SBJsonSample/README.md

```diff
@@ -0,0 +1,98 @@
+#SBJson5 Sample Application 
+
+##Setup
+This project uses SBJson5 as a dependency. When you first clone the repo if you have included ‘--recursive’ skip the next statement. Else navigate to WinObjC-samples root directory and run:
+```
+> git submodule update --init --recursive
+```
+
+Checkout the SBJson v5.0.0 version (SHA: c37ad93a2c1cd29c8a53aab67f4b30454ab03779)
+```
+> cd ThirdParty/SBJsonSample/SBJson-framework
+> git checkout v5.0.0
+```
+
+Then follow the steps from WinObjC (https://github.com/Microsoft/WinObjC/#getting-started-with-the-bridge) github repo documentation to create a visual studio solution.
+
+Expand the solution in the visual studio solution explorer. Expand SBJson->SBJson5_iOS(Universal Windows). right click on SBJson5_iOS(Universal Windows) and select unload project. Then scroll down to the page end.
+Now change the XML inside the ItemGroup tag to make necessary c headers to be public and importable in other plcaes of the sample code.
+
+change the  existing ItemGroup tag and its contents with :
```

Comment:
This was needed because SBJson5 is added as a git submodule and VSImporter generates the static library with its Headers which have "project" access modifiers that restricts some of the headers in static library to be used outside of the scope of library by default. To test these APIs I had to make those headers available across and beyond the static library container.